project-progression-policy.md

```diff
@@ -6,49 +6,62 @@ Confidential Computing Consortium (hereafter the "Consortium") via the
 a project may be admitted under and what the criteria and expectations are for a given
 stage, as well as the acceptance criteria for a project to move from one stage to
 another. It also describes the [Annual Review Process](#iv-annual-review-process)
-through which those changes will be evaluated and made. 
+through which those changes will be evaluated and made.
 
 Project progression - movement from one stage to another - allows projects to participate at the level that is most appropriate for them given where they are in their lifecycle. Regardless of stage, all Consortium projects benefit from a deepened alignment with existing projects, and access to mentorship, support, and Consortium resources.
 
-For more information about how your project can benefit from Consortium membership and services, please see [TBD Document]().
+### Benefits of being a recognized CCC project
 
+Here are ways your project can benefit by becoming a CCC recognized project?
+
+1. Recognition - a CCC project is recognized as meeting the goals of the CCC, namely protecting data-in-use.
+2. Community - the CCC is fostering a community of like minded members
+3. Participation - participate and lead in the ongoing development of the confidential computing paradigm. All projects are assigned a Mentor to assist in the integration of your project to the CCC and growth beyond that.
+4. Project Support - the CCC can provide limited funds to support project communication and infrastructure costs for CI/CD.
+5. Outreach - Part of the CCC's mission is to promote the use of confidential computing with various communities and hence its recognized projects to deliver that outcome.
 
 ## II. Project Proposal Process
 
 ### Introduction
 This governance policy sets forth the proposal process for projects to be accepted into the Consortium. The process is the same for both existing projects which seek to move into the Consortium and new projects to be formed within the Consortium.
 
-### Project Proposal Requirements
-Plese see the [Project Submission Template](https://github.com/confidential-computing/governance/blob/master/project-submission-template.md)
-
 ### Project Acceptance Process
-* Projects are required to present their proposal at a TAC meeting. Technical Projects
-  will specify an [OSI-approved license](https://opensource.org/licenses) for code contributions
-  and/or specification license structure as appropriate (e.g., Joint Development Foundation projects).
+The acceptance process is a three-stage process first a technical review with the Technical Advisory Committee (TAC) followed by a legal review and finally a vote at the Governing Board.
```

Comment:
My understanding is that the legal review and the TAC review can happen in either order.

Reply:
I think we should pick an order... it doesn't make sense to do a legal review if we're technically unhappy with the project.

Reply:
TAC Meeting 5/14 agreed that we would allow TAC and Legal processes to appear in parallel